cloudify_cli/constants.py

```diff
@@ -65,7 +68,8 @@
 CLOUDIFY_USERNAME_ENV = 'CLOUDIFY_USERNAME'
 CLOUDIFY_PASSWORD_ENV = 'CLOUDIFY_PASSWORD'
 
-CLOUDIFY_SSL_CERT = 'CLOUDIFY_SSL_CERT'
```

Comment:
This is a backwards incompatible change (changing the environment variable name). We should discuss if we want to maintain backwards compat here or not.
Anyway, the name of the environment variable should be `REST_CERT_PATH`